composer.json

```diff
@@ -22,7 +22,7 @@
         "illuminate/database": "5.6.*|5.7.*",
         "illuminate/filesystem": "5.6.*|5.7.*",
         "illuminate/support": "5.6.*|5.7.*",
-        "laravel/scout": "^6.1.1",
+        "laravel/scout": "^6.1.1|7.0.*",
```

Comment:
Thanks for contributing. You should add `"laravel/scout": "^6.1.1|^7.0". instead. Otherwise minor fixes will not get added.

Reply:
Oops. Sorry, missed that. It is now fixed 👍